.github/workflows/android.yml

```
@@ -29,18 +37,12 @@ jobs:
           name: Lint-Reports
           path: |
             app/build/reports/lint-results-debug.html
-            app/build/intermediates/lint_intermediate_text_report/debug/lintReportDebug/lint-results-debug.txt
-
-      - name: Run tests
-        env:
-          GITHUB_TOKEN: ${{ secrets.TRUSTWALLET_PAT }}
-          GITHUB_USER: ${{ secrets.TRUSTWALLET_USER }}
-        run: ./gradlew clean test
\ No newline at end of file
+            app/build/intermediates/lint_intermediate_text_report/debug/lintReportDebug/lint-results-debug.txt
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)